### PR TITLE
Typo fix in documentation.md

### PR DESCRIPTION
**Pull request by AdamDang**
Line 5: for it's CLI implementation->for its CLI implementation
Line 20: that these are ->that are

### CODE REVIEW

File: docs/development/documentation.md [hunk 1]
```diff
@@ -17,5 +17,5 @@ This command changes the cloud specification in the registry.
 It does not update the cloud resources, to apply the changes use "kops update cluster".
 ```
 
-* `Example`: Example(s) of how to use the command. This field is formatted as a code snippet in the docs, so make sure if you have comments that these are written as a bash comment (e.g. `# this is a comment`).
+* `Example`: Example(s) of how to use the command. This field is formatted as a code snippet in the docs, so make sure if you have comments that are written as a bash comment (e.g. `# this is a comment`).
```
Comment (justinsb) [inline docs/development/documentation.md]: I think this was correct previously.

**Comment by justinsb** [comment]
I think your "its" change was correct (thank you), but the other one I think was correct as-is.

**Comment by AdamDang** [comment]
OK, I've changed it back as before. Thanks!

**Comment by KashifSaadat** [comment]
Thanks @AdamDang. Please could you squash the commits down to one to keep the history cleaner?

**Comment by justinsb** [comment]
/approve
/lgtm

Just going to merge to clear the decks!

**Pull request merged**